src/Request/Read/BrowseDefinition.php

```diff
@@ -2,17 +2,15 @@
 
 namespace PhpTwinfield\Request\Read;
 
-use PhpTwinfield\Office;
-
 class BrowseDefinition extends Read
 {
     /**
      * Sets office and code if they are present.
      *
      * @param string $code
-     * @param Office|null $office
+     * @param $office
      */
-    public function __construct(string $code, Office $office = null)
```

Comment:
Please don't remove the typehint?

Reply:
Moving `setOffice` to `Read` gave errors in either `setOffice` in `Browse data`/`Transactions` or in most other objects because of correct type hinting in `Browse Data` and `Transactions` and bad type hinting in the other objects. Eventually chose the shortcut of removing typehints. Now properly fixed by fixing the bad type hinting